_episodes/04-scheduler.md

```diff
@@ -4,3 +4,26 @@ questions:
 - "How many and what kind of compute nodes are on this cluster?"
 - "How does the cluster get my tasks or programs to the compute nodes?"
 ---
+
+Topics: 
+
+Description of compute notes
+
+Description of what a scheduler does
+
+> ## Job scheduling roleplay
+> 
+> Divide the room in groups, with specific roles
+> 
+> * Pick one or two people to be the "scheduler"
+> * Have one third of the room be "users", given several slips of 
+> paper (or post-it notes) and pens
+> * Have two thirds of the room be "compute nodes"
+> 
+> "Users" should fill out their slips of paper with math problems.  
+> These math problems represent any kind of computation that you 
+> might want to run on the cluster.  How will these computations 
+> get to the compute nodes?  The "scheduler" will need to collect 
+> the math problems from the "users" and deliver them to the "compute 
+> nodes", returning the results when the node is done.  
```

Comment:
I tried this and we ended up using TONS of sticky notes; and also it mostly left everyone confused about what really was going on. I love this idea, but some more specific instructions to the instructor would be great :-)